src/lib/viewers/doc/docAssets.js

```diff
@@ -3,11 +3,16 @@ import { DOC_LEGACY_STATIC_ASSETS_VERSION, DOC_STATIC_ASSETS_VERSION } from '../
 
 const STATIC_VERSION = Browser.isIE() ? DOC_LEGACY_STATIC_ASSETS_VERSION : DOC_STATIC_ASSETS_VERSION;
```

Comment:
I realize it's not directly related to this change, but we can probably remove these legacy assets since we dropped support for IE11.